packages/core/src/binary/__test__/binary.test.ts

```diff
@@ -32,61 +30,20 @@ o.spec('CotarBinary.fake', () => {
   ];
   const TestFileSize = TestFiles.length + 1;
 
-  const tarIndexV1: Buffer = Buffer.alloc(TestFileSize * IndexV1RecordSize + IndexHeaderSize * 2);
+  // const tarIndexV1: Buffer = Buffer.alloc(TestFileSize * IndexV1RecordSize + IndexHeaderSize * 2);
```

Comment:
Comment out codes.

Reply:
Ill remove this in one of the future prs.